sources/packages/web/src/components/common/Reports.vue

```diff
@@ -56,6 +61,19 @@ export default defineComponent({
         props.reportList,
       );
     };
+    const formChanged = async (form: FormIOForm, event: any) => {
```

Comment:
We have some interfaces already like `FormIOChangeEvent` to be used.
Either way, unless really outside the scope of the PR, avoiding `any` must be tried.